pkg/config/utils.go

```diff
@@ -247,6 +248,83 @@ func checkConfig() error {
 	return nil
 }
 
+// processBasePath adjusts the current working dir by making it relative to the provided base path
+// NOTE: this function is not in use, we leave it here (in utils) in case we need it in the future
+func processBasePath() error {
```

Comment:
I wonder if the simple approach terraform took with `chdir` would be better? There's a lot of logic computing the absolute path. Is it necessary over the simple chdir?

cc: @Nuru

Reply:
https://www.terraform.io/cli/commands#switching-working-directory-with-chdir

Reply:
the function is not in use, we can delete it